problem_builder/answer.py

```diff
@@ -86,6 +90,14 @@ def get_model_object(self, name=None):
 
         return answer_data
 
+    def get_answer_data(self):
+        answer_data = self.get_model_object()
+        return {
+            "student_input": answer_data.student_input,
+            "created_on": answer_data.created_on,
+            "modified_on": answer_data.modified_on,
+        }
+
     @property
     def student_input(self):
```

Comment:
This method should be removed, I think - not sure why it's here, as it conflicts with https://github.com/open-craft/problem-builder/blob/de315bb040008554be3e509ae0fc82b094c15ff0/problem_builder/answer.py#L171-L188 which is more correct.